Creative/ATTNSDK.m

```diff
@@ -13,37 +13,34 @@ @implementation ATTNSDK {
     WKWebView *_webView;
     NSString *_domain;
     NSString *_mode;
-    NSString *_appUserId;
+    ATTNUserIdentity *_userIdentity;
 }
 
-
 - (id)initWithDomain:(NSString *)domain {
-    _domain = domain;
-    return [super init];
+    return [self initWithDomain:domain mode:@"production"];
 }
 
 - (id)initWithDomain:(NSString *)domain mode:(NSString *)mode {
     _domain = domain;
     _mode = mode;
+    _userIdentity = [[ATTNUserIdentity alloc] init];
     return [super init];
 }
 
-- (void)identify: (NSString *)appUserId {
-    _appUserId = appUserId;
+- (void)identify:(NSDictionary *)userIdentifiers {
```

Comment:
We will accept NSStrings here for back-compat, right? If so, doesn't have to be in this PR, just confirming

Reply:
yup! was going to add that in a new PR but ill actually just add here